README.md

```diff
@@ -147,6 +155,42 @@ Then, visit the URL logged to your terminal to view the now locally-running web
 
 You can stop the local PostgreSQL container server by running `docker-compose down`. If you wish to completely wipe the container database, including all the data added by the migrations, run `docker volume rm digital_marketplace_dm-vol`.
 
+#### Containerized
+
+##### Dependencies
+
+Ensure the following have been installed on your local machine:
+
+- Docker
+- Docker Compose
+- VS Code Docker extension
+
+To start the local development environment for the first time, run `docker-compose up` (this builds the containers, starts the database, runs the migrations, and starts the front- and back-ends in watch mode).
+
+The app will be available at localhost:3000 once you've seen the following among the terminal output:
+
+- `app_1 | Batch 1 run: 49 migrations`
+- `app_1 | [back-end] server started port="3000" host="0.0.0.0"`
+- `app_1 | Running "watch:frontEndStatic" (watch) task` (this is usually the last one to complete, so once you see it, you should be good to go)
+
+To start working within the development environment, open the VS Code Docker extension.
```

Comment:
I've added instructions for how to get into the containers via CLI, does that help? It's hard to get away from IDE-specific documentation because the functionality is somewhat IDE-specific--I never created shared volumes because the team decided to use the VS code docker extension and that doesn't require volumes. (Maybe other IDEs also have extensions that don't require volumes?)

I'm totally open to adding shared volumes at some point, but I think it's worth merging this readme into development now because it describes how to use what's already on the development branch.